fp/api/serializers.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import serializers
+
+from .models import Docket, Document
+
+class DocketSerializer(serializers.Serializer):
+    docket_no = serializers.CharField()
+    timestamp = serializers.ReadOnlyField()
```

Comment:
why is this a read-only field? Aren't you saving it in the model on POST request?

Reply:
because this field is auto generated and added through view that is handling the request and not posted by scrapy.